<launch>

    <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <!-- <arg name="world_name" value="$(find marmot)/worlds/marmot_world.world"/> -->
    <arg name="paused" value="false"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="true"/>
    <!-- <arg name="gui" value="false"/> -->
    <arg name="headless" value="false"/>
    <arg name="debug" value="false"/>
  </include>

  <!-- <node name="central_marmot" pkg="marmot" type="graphCentral.py" output="screen"/> -->


  <node pkg="map_server" type="map_server" name="map_broadcaster" args="$(find marmot)/map/house_map.yaml">
    <param name="frame_id" value="map" />
  </node>
  
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find marmot)/rviz/dynamic_rviz.rviz" />
</launch>